app/assets/stylesheets/modules/accordion-sections.scss

```diff
@@ -8,20 +8,20 @@
     padding-left: 0;
 
     .header {
-      float: left;
-      border-radius: 2px;
+      @extend .btn;
       color: $white;
-      display: inline-block;
       margin-right: 1em;
       cursor: pointer;
-      padding: 2px 10px;
```

Comment:
I'm noticing that some of these buttons when together do not pad

![screen shot 2018-11-02 at 11 39 54 am](https://user-images.githubusercontent.com/1656824/47931348-2f9acb80-de94-11e8-8846-16cc9e28dce1.png)